features/formatter_paths.feature

```diff
@@ -23,9 +23,10 @@ Feature: Formatter Paths
       <duration-stat>
       """
 
+  @wip
```

Comment:
Don’t forget to remove this tag before merging

Reply:
will do!

Reply:
done